tests/utils/test_manifest.py

```diff
@@ -348,8 +348,8 @@ def test_write_manifest(
 
         expected_metadata = {
             "schema": test_schema.model_dump_json(),
-            "partition-spec": test_spec.model_dump_json(),
-            "partition-spec-id": str(test_spec.spec_id),
+            "partition-spec": """[{"source-id":1,"field-id":1,"transform":"identity","name":"VendorID"},{"source-id":2,"field-id":2,"transform":"identity","name":"tpep_pickup_datetime"}]""",
```

Comment:
is it possible to not hardcode this value?

Reply:
I actually like that we are hardcoding this value because the issue wasn't caught because we inferred it from test_spec before :)

Reply:
I see, makes sense

Reply:
I prefer to hardcore the expected value so it is clear what is being returned when you go over the tests.